templates/src/blockchain/{{slug}}.ts.hbs

```diff
@@ -0,0 +1,38 @@
+import { get, post, EthBlock, EthTx, EthEstimateGas, EstimateGas, TransactionHash } from '@tatumio/tatum-core'
+import BigNumber from 'bignumber.js'
+
+/**
+ * For more details, see <a href="https://tatum.io/apidoc#operation/{{pascalCase slug}}Broadcast" target="_blank">Tatum API documentation</a>
+ */
+export const broadcast = async (txData: string, signatureId?: string): Promise<TransactionHash> =>
+  post(`/v3/{{slug}}/broadcast`, { txData, signatureId })
```

Comment:
This is fine as long as new API keeps this rule, I am saying it as I have noticed that this would not be compatible backwards as some chains use something different in some cases (eg ETH use `ethereum` instead of `eth`), but I guess it will not be used to generate already existing chains at all so this is all fine, I just wanted to point at the fact.

Reply:
👍